Sources/SmokeDynamoDB/AWSDynamoDBCompositePrimaryKeyTableGenerator.swift

```diff
@@ -0,0 +1,78 @@
+// Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+//  AWSDynamoDBCompositePrimaryKeyTableGenerator.swift
+//  SmokeDynamoDB
+//
+
+import Foundation
+import Logging
+import DynamoDBClient
+import DynamoDBModel
+import SmokeAWSCore
+import SmokeHTTPClient
+
+public class AWSDynamoDBCompositePrimaryKeyTableGenerator {
+    internal let dynamodbGenerator: AWSDynamoDBClientGenerator
```

Comment:
camel casing to `dynamoDbGenerator`

Reply:
I believe this is correct according to our readme to be consistent with the usage in AWS-

```
Naming Schema

For consistency in naming across the library, SmokeDynamoDB will case DynamoDB to what is observed and standardized in AWS's documentation of DynamoDB:

Lowercase: dynamodb

Use-cases: When used as a prefix to a function/variable name that is lower-cased or camel-cased.
Example:
dynamodbKeyWithPrefixedVersion
```